contracts/cw1-whitelist-ng/Cargo.toml

```diff
@@ -25,17 +24,20 @@ multitest = ["cw-multi-test", "anyhow"]
 cw-utils = { path = "../../packages/utils", version = "0.13.2" }
 cw1 = { path = "../../packages/cw1", version = "0.13.2" }
 cw2 = { path = "../../packages/cw2", version = "0.13.2" }
-cosmwasm-std = { version = "1.0.0-beta8", features = ["staking"] }
+cosmwasm-std = { version = "1", features = ["staking"] }
 cw-storage-plus = { path = "../../packages/storage-plus", version = "0.13.2" }
 schemars = "0.8.1"
 serde = { version = "1.0.103", default-features = false, features = ["derive"] }
 thiserror = { version = "1.0.23" }
 cw-multi-test = { path = "../../packages/multi-test", version = "0.13.2", optional = true }
 anyhow = { version = "1", optional = true }
+cw-derive = { version = "0.13.2", path = "../../packages/cw-derive" }
+serde-cw-value = { version = "0.7.0", git = "https://github.com/hashedone/serde-cw-value.git" }
```

Comment:
👀 you don't need patch directive?

Reply:
I don't understand? I can use git normally, even if tag, or slug. I mean I need to correct the git repo as I gave ownership to CosmWasm today, but on the moment of pushing it it was correct ;)

Reply:
I thought this was something indirectly imported via serde, not directly here.
Nevermind me, tired and confused about package imports.

Reply:
Ah, no - we need it. And it wont be here finally, long story - I can talk about details in Malaga ;)